client/src/Landing/Landing.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './css/Landing.css';
+import CorndelLogo from './img/Corndel-Logo.png'
+import placeholders from './usernames.js';
+
+export default class Landing extends React.Component {
+	constructor(props){
+		super(props)
+		this.state = {username: ""}
+	}
+
+	componentDidMount(){
+		let ranName = placeholders[Math.floor(Math.random() * placeholders.length)]
+		document.getElementById('username').placeholder = ranName.name;
```

Comment:
I know I suggested doing it this way but I actually think a nicer way would be to work out the randomName in the constructor, and save it as `this.placeHolderName= \\whatever` and then in the render function you can put your `<input placeHolder={this.placeHolderName} />`

Reply:
and then you don't need the componentDidMount. Should be (fractionally) faster as it doesn't require 2 renders